sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -472,11 +472,11 @@ identifierComment
     ;
 
 relationPrimary
-    : tableIdentifier sample? (AS? strictIdentifier)?               #tableName
-    | '(' queryNoWith ')' sample? (AS? strictIdentifier)?           #aliasedQuery
-    | '(' relation ')' sample? (AS? strictIdentifier)?              #aliasedRelation
-    | inlineTable                                                   #inlineTableDefault2
-    | identifier '(' (expression (',' expression)*)? ')'            #tableValuedFunction
+    : tableIdentifier sample? (AS? strictIdentifier)?                                        #tableName
+    | '(' queryNoWith ')' sample? (AS? strictIdentifier)?                                    #aliasedQuery
+    | '(' relation ')' sample? (AS? strictIdentifier)?                                       #aliasedRelation
+    | inlineTable                                                                            #inlineTableDefault2
+    | identifier '(' (expression (',' expression)*)? ')' (AS? identifier identifierList?)?   #tableValuedFunction
```

Comment:
Then, I'll update this pr to separate this rule and share it with the inline table rule.